client/js/s3/request-signer.worker-manager.js

```diff
@@ -0,0 +1,88 @@
+/* globals qq, CryptoJS */
+
+/**
+ * Manages creation and communication of s3 signature workers.
+ */
+qq.s3.RequestSignerWorkerManager = function (o) {
+    "use strict";
+    var _worker = null,
+        _workerPromises = {},
+        options = {
+            workerUrl: null,
+            log: function(str, level) {}
+        };
+    qq.extend(options, o, true);
+
+    function init() {
+        var workerUrl;
+        switch (typeof options.workerUrl) {
+            case "string":
+                if (options.workerUrl !== "inline") {
+                    workerUrl = options.workerUrl;
+                } else {
+                    if (!qq.s3.createS3InlineWorkerUrl) {
```

Comment:
Left over from when s3 core didn't include the inline worker. It should be safe to remove it.